storage/libradb/src/libradb_test.rs

```diff
@@ -454,4 +455,38 @@ proptest! {
         assert!(account_state_with_proof.blob.is_none());
         assert!(events.is_empty());
     }
+
+    #[test]
+    fn test_get_from_non_existent_event_stream(
```

Comment:
It feels like this belongs to another commit. If this is about DISCOVERY_SET_CHANGE_EVENT maybe rename a bit.